src/styles/block/main1.scss

```
@@ -0,0 +1,165 @@
+body,
+p,
+h2 {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+}
+
+.container {
+  row-gap: 46px;
+  column-gap: 48px;
+  display: grid;
+  grid-template-columns: repeat(1,200px);
+  padding: 50px;
+  justify-content: center;
+}
+
+@media (min-width:488px) {
+  .container {
+    grid-template-columns: repeat(2,200px);
+  }
+}
+
+@media (min-width:768px) {
+  .container {
+    grid-template-columns: repeat(3,200px);
+  }
+}
+
+@media (min-width:1024px) {
+  .container {
+    grid-template-columns: repeat(4,200px);
+  }
+}
+
+.card {
+  border-radius: 5px;
+  background-color: #{$white-color-primary};
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  &:hover {
+    transition-duration: 300ms;
+    transform: scale(1.2);
+  }
```

Transition should be here to work on both side (hover in/ hover out) 
```suggestion
  flex-direction: column;
  padding: 32px 16px 16px;
  transition: propertie duration

  &:hover {
    transform: scale(1.2);
  }
```